README.md

```diff
@@ -119,5 +119,4 @@ Contributors are Welcome. Please read the [Contribution](https://github.com/mbj3
 
 - [Appnotion](https://github.com/helloguille)
 - [MarcoHengstenberg (Marco Hengstenberg)](https://github.com/MarcoHengstenberg)
-- [rusrushal13 (Rushal Verma)] (https://github.com/rusrushal13/)
```

Comment:
Not really typo per se, I think this was not properly linked with space in-between the name and the GitHub link.